src/main/java/de/tum/in/www1/artemis/repository/UserRepository.java

```diff
@@ -114,6 +114,12 @@ OR lower(user.login) = lower(:#{#searchInput}))
     @Query("SELECT user FROM User user WHERE user.isDeleted = false AND :#{#groupName} MEMBER OF user.groups")
     List<User> findAllInGroup(@Param("groupName") String groupName);
 
+    @Query("""
+            SELECT user from User user
+            JOIN user.groups
+            """)
+    Set<User> findAllInAnyGroup();
+
```

Comment:
Let's add a warning here that this is only supposed to be called from test classes. Calling this on production doesn't sound like a good idea. :D

Reply:
I suggest to move this method into a test repository then

Reply:
We moved the method to a test repository